src/NextjsDistribution.ts

```diff
@@ -338,8 +342,8 @@ export class NextjsDistribution extends Construct {
     //   - if 403, fall back to lambda handler (mostly for /)
     //   - if 404, fall back to lambda handler
     const fallbackOriginGroup = new origins.OriginGroup({
-      primaryOrigin: s3Origin,
-      fallbackOrigin: serverFunctionOrigin,
+      primaryOrigin: serverFunctionOrigin,
```

Comment:
Since static assets are cache via bucket cache policy and static pages are now cached via headers, we should always try to hit the lambda first to save ~300ms in response time.
It is better to give faster page response time over asset response time anyways.